cryptoki/src/object.rs

```diff
@@ -14,7 +14,7 @@ use std::fmt::Formatter;
 use std::mem::size_of;
 use std::ops::Deref;
 
-const MAX_CU_ULONG: CK_ULONG = !0;
+pub(crate) const MAX_CU_ULONG: CK_ULONG = !0;
```

Comment:
It's not your doing but I am wondering why this is not called `MAX_CK_ULONG` :thinking: ?